testdata/src/go.uber.org/annotationparse/annotationparse.go

```diff
@@ -46,7 +46,7 @@ func foo(a, b *bar, c *bar, d, e *bar) (f, g *bar, h *bar) {
 	myBar.jar = a
 	myBar.karp = a
 	myBar.lug = a
-	myBar.myr = a //want "nilable value assigned into the field `myr`"
+	myBar.myr = a //want "assigned into the field `myr`"
```

Comment:
I am curious, how do the full error messages look like here on no-infer mode? Presumably they will be a two locations flow (from directly NILABLE to directly NONNIL, right?). Could we get an example into the PR description?

Reply:
Now with the producer + consumer concatenation, it will be a single line. Added an example in the PR description.

Reply:
👍